src/main.js

```diff
@@ -116,9 +116,9 @@ var mainImageURL = document.querySelector('.poster-img')
 var mainQuote = document.querySelector('.poster-quote')
 
 var posterGrid = document.querySelector('.saved-posters-grid')
-var displaySavedPosters = document.createElement('article')
-
-posterGrid.appendChild(displaySavedPosters('h2'))
```

Comment:
THANKS FOR THIS we should have tested the site after adding this but we didn't. Lesson learned.